looper/looper.py

```diff
@@ -395,17 +404,19 @@ def __call__(self, args, rerun=False, **compute_kwargs):
                 )
 
         submission_conductors = {}
+
         for piface in self.prj.pipeline_interfaces:
             conductor = SubmissionConductor(
                 pipeline_interface=piface,
                 prj=self.prj,
                 compute_variables=comp_vars,
-                delay=args.run.time_delay,
-                extra_args=args.run.command_extra,
-                extra_args_override=args.run.command_extra_override,
-                ignore_flags=args.run.ignore_flags,
-                max_cmds=args.run.lumpn,
-                max_size=args.run.lump,
+                delay=getattr(args, "time_delay", None),
+                extra_args=getattr(args, "command_extra", None),
+                extra_args_override=getattr(args, "command_extra_override", None),
+                ignore_flags=getattr(args, "ignore_flags", None),
+                max_cmds=getattr(args, "lumpn", None),
```

Comment:
didn't these get changed to `lump-n` -style names? so they would be `lump_n`, etc, in Python?

Reply:
I flipped them back to be --lump-n, and --lump-j as thy were previously.